src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -0,0 +1,25 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+// ------------------------------------------------------------------------------
+// Changes to this file must follow the http://aka.ms/api-review process.
+// ------------------------------------------------------------------------------
+
+namespace System.Text.Json
+{
+    public enum JsonTokenType : byte
```

Comment:
I think he is just seeding the library and then will add code progressively. Just so that the upstream plan with the package and what platforms it is going to support was clear.